packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2750,6 +2750,10 @@ class RenderMouseRegion extends RenderProxyBox {
   @visibleForTesting
   MouseTrackerAnnotation get hoverAnnotation => _hoverAnnotation;
 
+  Matrix4 _eventTransformer() {
+    return Matrix4.inverted(getTransformTo(null));
```

Comment:
Same comment here that I also left on https://github.com/flutter/flutter/pull/48271